ref/general/packaging-deploying.adoc

```diff
@@ -14,79 +14,69 @@
-The `minify` option specifies that the runnable JAR file contains only what you need to run your application so the application is as lightweight as possible.
-
-== Containerize
-After you create a server package by packaging your application with the Open Liberty runtime, you can containerize it for deployment.
+This local packaging method can be useful for debugging and testing during development.
+
+To package into a runnable JAR file, specify `runnable` as the package type in the `include` parameter or attribute.
+You can also specify the `minify` option in the `include` property or attribute so that your runnable JAR file contains only what you need to run your application.
+
+== Containerization
+After you package your application into a server package with `usr` contents and build it with Maven or Gradle, you can containerize it for cloud deployment.
 Containerizing your server package bundles together the environment that your application needs to run.
 With Docker containers, you have the portability to run your application on any machine that is running Docker.
 
 A Docker image is a multi-layered file that serves as a template for a Docker container.
 The simplest way to containerize your server package is by using the latest link:https://hub.docker.com/_/open-liberty[Open Liberty Docker images], which include the most recent feature versions and link:/docs/ref/general/#security-vulnerabilities.html[security fixes].
-If you want to build your own Docker image, you need to create a Dockerfile.
+If you want to build your own Docker image, you need to create a link:https://docs.docker.com/engine/reference/builder/[Dockerfile].
 A Dockerfile is a text file that contains commands about how an image is built.
 If you choose to create your own Docker image, make sure to follow link:/docs/ref/general/#server-configuration-hardening.html[server configuration hardening best practices].
 
 Once you have a Dockerfile, you can run the `docker build` command to build an image for the container.
 Finally, the `docker run` command runs your container that houses the application.
 
-To learn more about running applications with Open Liberty Docker images, see these guides on link:https://openliberty.io/guides/docker.html[Using Docker containers to develop microservices] and link:/guides/containerize.html[Containerizing microservices].
+To learn more about running applications with Open Liberty Docker images, see this guides on link:/guides/containerize.html[Containerizing microservices].
 
-== Deploy
-After you containerize your server package, you can deploy it to link:https://kubernetes.io/[Kubernetes] or link:https://www.openshift.com/[OpenShift].
+== Deployment
+You can deploy web applications and enterprise applications on Open Liberty.
+Deploy containerized applications to the cloud, or, for traditional environments, deploy a WAR deployment artifact on Open Liberty.
```

Comment:
I'm having trouble with this sentence. It reads like an instruction, but then there aren't any actual instructions.